src/client/Microsoft.Identity.Client/Platforms/Android/AuthenticationContinuationHelper.cs

```diff
@@ -51,7 +51,7 @@ public static void SetAuthenticationContinuationEventArgs(int requestCode, Resul
                 //The BrokerRequestId is an ID that is attached to the activity launch during brokered authentication
                 // that indicates that the response returned to this class is for the broker.
                 LastRequestLogger?.Info("Processing result from broker.");
-                AndroidBroker.SetBrokerResult(data, (int)resultCode, LastRequestLogger);
+                AndroidBrokerStaticHelper.SetBrokerResult(data, (int)resultCode, LastRequestLogger);
```

Comment:
I think that Result enum is misleading because of Xamarin translations. You'd think it can only have values -1, 0 and 1 - 
![image](https://user-images.githubusercontent.com/12273384/107390735-e2d0e100-6aef-11eb-8d1b-de1cb4081ec3.png)

But this is not the case, Authenticator can set it to other values. Custom activities (which Android broker is) can set it to different results. 

In fact, the Java API uses an int:

https://developer.android.com/reference/android/app/Activity#onActivityResult(int,%20int,%20android.content.Intent)